res/database/Default/user/stukov_scripts_adaptations.xml

```diff
@@ -131,67 +152,81 @@
 			<title>
 				<de>Die Goblins</de>
 				<en>The Goblins</en>
+				<pl>Gobliny</pl>
 			</title>
 			<title_original>
 				<de>Die Goblins</de>
 				<en>The Goblins</en>
+				<pl>Gobliny</pl>
 			</title_original>
 			<description>
 				<de>Verfilmung der Buchreihe Die Goblins um %Goblin% den Goblin und den vergessenen Gott %Gott%.</de>
 				<en>Adaptation of the book series The Goblins about %Goblin% the goblin and the forgotten god %Gott%.</en>
+				<pl>Adaptacja serii książek The Goblins o %Goblinie% goblinie i zapomnianym bogu %Gott%.</pl>
```

Comment:
variable %Goblin% must not be changed to %Goblinie%

Reply:
obviously. corrected.